demo/vl-proza-message.html

```diff
@@ -4,6 +4,26 @@
         <title>vl-proza-message voorbeeld</title>
         <script type="module" src="demo.js"></script>
         <script type="module" src="../vl-proza-message.src.js"></script>
+        <script src="../node_modules/fetch-mock/es5/client-bundle.js"></script>
+
+        <script>
+            fetchMock.mock('proza/domein/foo/bar', {
```

Comment:
Eventueel nog eens kijken of we fetch-mock ook kunnen gebruiken voor de unit testen.